help/en/package/jmri/jmrit/operations/Operations.shtml

```diff
@@ -4180,6 +4185,22 @@
    <a name="RouteTools" id="RouteTools"></a>
    <h3>Route Tools</h3>
 
+   <a name="RouteBlockingOrder" id="RouteBlockingOrder"></a>
+   <h4>Blocking Order</h4>
+   <p>
+    The program when creating a <a href="#PrintingManifests">Manifest</a> normally blocks
+    cars in the train by their destination. Cars for the first destination in a train's route are
+    placed immediately after the locomotives, followed by the next location's set of cars,
+    continuing until the last set of cars which can be found at the end of the Manifest's for the
+    train's departure location. For example, you might want to have the cars for a location placed
```

Comment:
It seems something is missing in this text. First, you describe the old way of doing things, there the program places the cars in the order by destination.
This is immediately followed by the sentense "For example, you might want to have the cars for a location placed..." which seems to refer to the new way to do things, to have a different blocking order. So the text is a bit awkward to me.

A suggestion is to have something like this:

The program when creating a Manifest normally blocks
cars in the train by their destination. Cars for the first destination in a train's route are
placed immediately after the locomotives, followed by the next location's set of cars,
continuing until the last set of cars which can be found at the end of the Manifest's for the
train's departure location.
**But with this tool, you can have a different blocking order.** For example, you might want
to have the cars for a location placed ...